code/modules/lighting/lighting_atom.dm

```diff
@@ -7,34 +7,39 @@
 	var/tmp/list/light_sources       // Any light sources that are "inside" of us, for example, if src here was a mob that's carrying a flashlight, that flashlight's light source would be part of this list.
 
 // The proc you should always use to set the light of this atom.
-/atom/proc/set_light(l_range, l_power, l_color)
-	if(l_power != null) light_power = l_power
-	if(l_range != null) light_range = l_range
-	if(l_color != null) light_color = l_color
+/atom/proc/set_light(var/l_range, var/l_power, var/l_color)
+	if (l_power != null)
+		light_power = l_power
+
+	if (l_range != null)
+		light_range = l_range
+
+	if (l_color != null)
+		light_color = l_color
 
 	update_light()
 
 // Will update the light (duh).
 // Creates or destroys it if needed, makes it update values, makes sure it's got the correct source turf...
 /atom/proc/update_light()
 	set waitfor = FALSE
-	if(gcDestroyed)
+	if (gcDestroyed)
```

Comment:
Then stop formatting code that you didn't touch. I'm tired of you and dylan doing this: it bloats PRs and history with needless changes that someone else might well revert for different formatting reasons and then we make no progress on actual code.